examples/unattended_baggage/demo.ipynb

```diff
@@ -0,0 +1,330 @@
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 2: Query on baggage's owner\n",
+    "\n",
+    "To filter baggage that doesn't have a owner for some time period (i.e. `owner` is `None`), we use `vqpy.utils.continuing` with condition `lambda x: x is None` on property `\"owner\"`. Duration is set to `10` (seconds), and we use the name `no_owner` to describe the filter on `owner`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filter_cons = {\n",
+    "    \"__class__\": lambda x: x == Baggage,\n",
+    "    \"owner\": vqpy.utils.continuing(\n",
+    "        condition=lambda x: x is None, duration=10, name=\"no_owner\"\n",
+    "    ),\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For each baggage without owner, we select the following for outputting:\n",
+    "\n",
+    "- `track_id`, tracking id, built-in property of VObj\n",
+    "- `tlbr`, bounding box, also built-in property of VObj. Use post-processing function to converted to string before serializing.\n",
+    "- `no_owner_periods`, a list of time periods (in seconds) of `Baggage` having no owner\n",
+    "\n",
+    "\tThis property is added to `Baggage` by wrapper `vqpy.utils.continuing` in `filter_cons`, its name from `{name}_periods` where `name` \"no_owner\" is defined in the wrapper"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "select_cons = {\n",
+    "    \"track_id\": None,\n",
+    "    \"tlbr\": lambda x: str(x),\n",
+    "    \"no_owner_periods\": None,\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The query is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class FindUnattendedBaggage(vqpy.QueryBase):\n",
+    "    @staticmethod\n",
+    "    def setting() -> vqpy.VObjConstraint:\n",
+    "        return vqpy.VObjConstraint(\n",
+    "            filter_cons=filter_cons,\n",
+    "            select_cons=select_cons,\n",
+    "            filename=\"unattended_baggage\",\n",
+    "        )"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 3: Running the query"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the `Person` VObj and the query defined, we can run the query, where:\n",
+    "\n",
+    "- `cls_name` is a tuple for mapping numerical outputs of object detector to str\n",
+    "\n",
+    "\te.g. the built-in `vqpy.COCO_CLASSES` here starts with `(\"person\", \"bicycle\", ...)`, meaning that we will map output `0` of object detector to COCO class `\"person\"`\n",
```

Comment:
which is the detection class name. Here, we use `COCO_CLASSES` since it includes all the class names we are interested in the unattended baggage query, i.e. "person", "backpack", and "suitcase". And we can then specify the mapping from the COCO class names to our interested `VObj`s with `cls_type`

Reply:
Changed